demos/signal-processing/Dockerfile-nifi

```diff
@@ -1,3 +1,5 @@
+# docker build -f ./Dockerfile-nifi -t docker.stackable.tech/demos/nifi:1.27.0-postgresql .
+
```

Comment:
note: This comment is not needed anymore, as the image gets built by CI, see https://github.com/stackabletech/demos/blob/main/.github/workflows/dev_nifi.yaml

Reply:
Removed in 7649d89